libcontainer/cgroups/stats.go

```diff
@@ -103,8 +103,22 @@ type MemoryStats struct {
 	// if true, memory usage is accounted for throughout a hierarchy of cgroups.
 	UseHierarchy bool `json:"use_hierarchy"`
 
-	Stats map[string]uint64 `json:"stats,omitempty"`
-	PSI   *PSIStats         `json:"psi,omitempty"`
+	Stats      map[string]uint64 `json:"stats,omitempty"`
+	PSI        *PSIStats         `json:"psi,omitempty"`
+	EventCount MemoryEventsCount `json:"events_count,omitempty"`
+}
+
+type MemoryEventsCount struct {
+	// count of memory reclaim (when usage is under the low boundary)
+	ReclaimLowCount uint64 `json:"reclaim_low_count"`
```

Comment:
I think we can remove the suffix `Count` for all fields in this struct, because the struct's meaning is `memory events count`.
Another thing: s/MemoryEventsCount/MemoryEventCount/